EnergyCorrelatorOptions.h

```diff
@@ -63,6 +65,7 @@ namespace EnergyCorrelatorOptions {
   const float theta = 0;
   const float pTSmear = 0.02;
   const float effVal = 1;
+  const float jetPtSmear = 0.02;
```

Comment:
I'd suggest also adding a flag to turn smearing on/off (like you did with the constituent-smearing). It makes it more explicit when smearing is being applied vs. not!